var/spack/repos/builtin/packages/ccache/package.py

```diff
@@ -0,0 +1,45 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Ccache(AutotoolsPackage):
+    """ccache is a compiler cache. It speeds up recompilation by caching
+    previous compilations and detecting when the same compilation is being done
+    again."""
+
+    homepage = "https://ccache.samba.org/"
+    url      = "https://www.samba.org/ftp/ccache/ccache-3.3.3.tar.gz"
+
+    version('3.3.3', 'ea1f95303749b9ac136c617d1b333eef')
+    version('3.3.2', 'b966d56603e1fad2bac22930e5f01830')
+    version('3.3.1', '7102ef024cff09d353b3f4c48379b40b')
+    version('3.3'  , 'b7ac8fdd556f93831618483325fbb1ef')
+    version('3.2.9', '8f3f6e15e75a0e6020166927d41bd0b3')
+
+    depends_on('autoconf', type='build')
```

Comment:
Indeed, it is commented out. My apologies. 

But I'm now confused. I'm not an expert on build systems (least of all `autotools`), so I now don't know what build dependencies (if any at all) I actually need for this package. I removed the line `depends_on('autoconf', type='build')` and it still installs fine. But looking at the first phase of the build process suggests that I still need this line? At least that's how I interpret @citibeth's comment.
```bash
==> Building ccache [AutotoolsPackage]
==> Executing phase : 'autoreconf'
==> Executing phase : 'configure'
==> Executing phase : 'build'
==> Executing phase : 'install'
==> Successfully installed ccache
```